<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局设置*/
* {	font-family: &quot;微软雅黑&quot;; font-size: 15px;}
QWidget#Window{
   background:#0AA0F2;
	outline:none;
	color:#333;

}
/*最上边的背景*/
QWidget#Status{
	background: #569082;
	outline:none;
	color:#333;
}
/*后台运行按钮*/
QPushButton#btnHide {
	background: transparent;
	border: none;
	color: #ffffff;
	font-size: 18px;
}
QPushButton#btnHide:hover {
	background: #569000;
}
QPushButton#btnHide:pressed {
	background: #569040;
}
/*全屏运行按钮*/
/*后台运行按钮*/
QPushButton#btnMAX {
	background: transparent;
	border: none;
	color: #ffffff;
	font-size: 18px;
}
QPushButton#btnMAX:hover {
	background: #569000;
}
QPushButton#btnMAX:pressed {
	background: #569040;
}

/*提示栏*/
QLabel#Notice{
background: #569000;
       /*background: transparent ;*/
	color: #ffffff;
	font-size: 18px;
}

/*三个重要的按钮*/
/*开始服务按钮*/
QPushButton#btnStart{
     background-color: red;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnStart:hover {
     background-color: rgb(220, 124, 0);
     border-style: inset;
}
QPushButton#btnStart:pressed {
     background-color: rgb(224, 0, 0);
     border-style: inset;
}

/*停止服务按钮*/
QPushButton#btnStop{
     background-color: red;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnStop:hover {
     background-color: rgb(220, 124, 0);
     border-style: inset;
}
QPushButton#btnStop:pressed {
     background-color: rgb(224, 0, 0);
	 border-style: inset;
} 


/*退出按钮*/
QPushButton#btnQuit{
     background-color: red;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
    font: bold 15px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnQuit:hover {
     background-color: rgb(220, 124, 0);
     border-style: inset;
}
QPushButton#btnQuit:pressed {
     background-color: rgb(224, 0, 0);
     border-style: inset;
}
/*四个按钮*/
/*背景*/
QWidget#MainWork{
  background-color: rgb(0, 10, 200);
}
/*单独消息*/
QPushButton#btnPrivate{
     background-color: rgb(199, 180, 200);
     border-radius: 3px;
     border-color: beige;
   font: bold 15px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnPrivate::hover{
     background-color: rgb(90, 180, 200);
}
QPushButton#btnPrivate:pressed{
     background-color: rgb(199, 100, 100);
 	 border-style: inset;
}
/*发送公告*/
QPushButton#btnPublic{
     background-color: rgb(199, 180, 200);
     border-radius: 5px;
     border-color: beige;
   font: bold 15px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnPublic::hover{
     background-color: rgb(90, 180, 200);
}
QPushButton#btnPublic:pressed{
     background-color: rgb(199, 100, 100);
 	 border-style: inset;
}
/*显示压力*/
QPushButton#btnPressure{
     background-color: rgb(199, 180, 200);
     border-radius: 5px;
     border-color: beige;
    font: bold 15px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnPressure::hover{
     background-color: rgb(90, 180, 200);
}
QPushButton#btnPressure:pressed{
     background-color: rgb(199, 100, 100);
 	 border-style: inset;
}
/*监控聊天*/
QPushButton#btnWatch{
     background-color: rgb(199, 180, 200);
     border-radius: 5px;
     border-color: beige;
     font: bold 15px;
     min-width: 10em;
     padding: 6px;
}
QPushButton#btnWatch::hover{
     background-color: rgb(90, 180, 200);
}
QPushButton#btnWatch:pressed{
     background-color: rgb(199, 100, 100);
 	 border-style: inset;
}
/*发送面板*/
QWidget#MyEdit{
  background-color: rgb(78, 10, 200);
}	
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="VL2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="Status" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnHide">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HS">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_live">
            <property name="text">
             <string>当前在线人数：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HS3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_Connect">
            <property name="text">
             <string>链接人数:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="HS2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnMAX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>#</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="MainOption" native="true">
       <layout class="QVBoxLayout" name="Main">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="Notice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>这里是通知消息</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="HL3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnStart">
            <property name="text">
             <string>开始服务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStop">
            <property name="text">
             <string>停止服务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuit">
            <property name="text">
             <string>退出系统</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="People" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="listView"/>
        </item>
        <item>
         <widget class="QWidget" name="showPanel" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="MainWork" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="HL4">
        <item>
         <widget class="QPushButton" name="btnPrivate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>单独消息</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPublic">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>发送公告</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPressure">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>显示压力</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWatch">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>监控聊天</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="MyEdit" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QWidget" name="MyEditAreaPanel" native="true">
          <layout class="QHBoxLayout" name="MyEditArea">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <widget class="QTextEdit" name="textEditMessage">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSend">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="MyEditToolsPanel" native="true">
          <layout class="QHBoxLayout" name="MyTextTools">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_sendTo">
             <property name="text">
              <string>发送给：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbSendToID"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>T1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolBtn2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>T2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolBtn3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>T3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolBtn4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>T4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolBtn5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>T5</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="HS11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
